<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpressmanView</class>
 <widget class="QWidget" name="ExpressmanView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>快递员界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabPackage">
      <attribute name="title">
       <string>包裹管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabProfile">
      <attribute name="title">
       <string>个人信息维护</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 